doc/developers/build_app.rst

```diff
@@ -73,3 +73,18 @@ Additionally, the `cchost` binary must be told that the enclave type is debug:
 .. code-block:: bash
 
     $ cchost --enclave-file liblua_generic.enclave.so.debuggable --enclave-type debug [args]
+
+Build Container
+---------------
+
+With every release of CCF, a base build container is provided to facilitate reproducible builds and continuous integration.
+It contains everything needed to build and test CCF applications.
+
+.. literalinclude:: ../../docker/app_ci
```

Comment:
Is there much value including the `Dockerfile` here?

Reply:
I think it’s useful to have a reference to it if you want to tweak it, I’ll see if I can change it to a link somehow.

Reply:
Can't find a nice way to do that. It's easy for Python code, but not for arbitrary files.